### ISSUE TITLE
[API] Admin Panel Controls

### ISSUE THREAD

**Issue by HaffCaff** [Rapid-Reservation/rapid-api#32]
Implement the following admin control endpoints:
- endpoint to reset tables:  Blocked by: https://github.com/Rapid-Reservation/rapid-api/issues/14
- endpoint to make a user an admin
- endpoint to remove admin

**Comment by HaffCaff** [comment]
Out of scope
